algorithms/merging/merge.py

```diff
@@ -237,7 +244,7 @@ def merge_and_truncate(params, experiments, reflections):
                 params.merging.use_internal_variance,
             )
         except DialsMergingStatisticsError as e:
-            logger.info(e)
+            logger.error(e)
```

Comment:
```suggestion
            logger.error(e, exc_info=True)
```
pass on trace information

Reply:
Thanks, I didn't know you could do that.